detections/endpoint/windows_system_remote_discovery_with_query.yml

```
@@ -0,0 +1,57 @@
+name: Windows System Remote Discovery With Query
+id: 94859172-a521-474f-97ac-4cf4b09634a3
+version: 1
+date: '2025-01-06'
+author: Steven Dick
+status: production
+type: TTP

```

Having query simply execute is not TTP worthy imo.

```suggestion
type: Anomaly

```

---

Could vary depending on environment, but I have no issues setting it to RBA only.